components/frontend/src/issue/IssueStatus.js

```diff
@@ -119,7 +119,7 @@ prefixName.propType = {
 }
 
 function issueLabel(issueStatus, settings, error) {
-    const color = error ? "red" : { todo: "grey", doing: "blue", done: "green" }[issueStatus.status_category ?? "todo"]
+    const color = error ? "red" : ISSUE_STATUS_COLORS[issueStatus.status_category ?? "unknown"]
```

Comment:
What does colour "unknown" mean? Why aren't unknown and error part of the `ISSUE_STATUS_COLORS`?

Reply:
`ISSUE_STATUS_COLORS` does include "unknown" now. Unknown means the issue status is unknown, because the status hasn't been collected yet or the connection with the issue tracker failed. I'll add a comment.